backend/src/server/routes/v2/project-router.ts

```diff
@@ -214,7 +214,7 @@ export const registerProjectRouter = async (server: FastifyZodProvider) => {
         slug: slugSchema.describe("The slug of the project to delete.")
       }),
       response: {
-        200: ProjectsSchema
+        200: SanitizedProjectSchema
```

Comment:
Not a new naming convention rather it was there before. I had to remove some fields getting exposed from project schema because project table now contains encrypted data keys